lib/utils/resource-links.js

```diff
@@ -0,0 +1,60 @@
+function isResourceLink (field) {
+  return field && field.sys && field.sys.type === 'ResourceLink'
+}
```

Comment:
I would make this check a bit more resilient to prevent bugs when the customer is relying on this SDK version and we release new `ResourceLink` types.
```suggestion
function isContentfulEntryResourceLink (field) {
  return field && field.sys && field.sys.type === 'ResourceLink' && field.sys.linkType === 'Contentful:Entry'
}
```